packages/expo-modules-core/ios/Core/Views/SwiftUI/SwiftUIViewDefinition.swift

```diff
@@ -25,6 +25,40 @@ public extension ExpoSwiftUIView {
     }
   }
 
+  /**
+   Returns React's children as SwiftUI views, with any nested HostingViews stripped out.
+   */
+  func UnwrappedChildren<T: View>( // swiftlint:disable:this identifier_name
+    @ViewBuilder transform: @escaping (_ child: AnyView, _ isHostingView: Bool)
+    -> T
+  ) -> some View {
+    guard let children = props.children else {
+      return AnyView(EmptyView())
+    }
+    let childrenArray = Array(children)
+    let mockShadowNodeProxy = {
+      let proxy = ExpoSwiftUI.ShadowNodeProxy()
+      proxy.setViewSize = nil
+      return proxy
+    }()
```

Comment:
mocking this shadowNodeProxy since we want it to render naturally within its SwiftUI parent